package.json

```diff
@@ -37,7 +38,8 @@
   "author": "Lawrence Gosset <lawrence.gosset@onfido.com> (https://github.com/gosseti)",
   "devDependencies": {
     "@babel/cli": "^7.0.0",
-    "@babel/core": "^7.0.0",
+    "@babel/core": "^7.1.2",
+    "babel-core": "^7.0.0-bridge.0",
```

Comment:
@stefaniacardenas do you know on which PR this was added? I would like to take a look a little more on the context of this change

Reply:
The one that introduced Jest https://github.com/onfido/onfido-sdk-ui/pull/483/files